app/views/smart_proxies/plugins/_puppet_ca.html.erb

```diff
@@ -0,0 +1,27 @@
+<h3><%= _("Puppet Certificate Authority") %></h3>
+<%= show_feature_version('puppetca') %>
```

Comment:
I've added http://projects.theforeman.org/issues/12948 for a robust way of knowing the ca info, but however, for now, a simple sort for oldest cert would suffice (as the CA is the first certificate created).